crates/eips/src/eip4895.rs

```diff
@@ -37,6 +38,63 @@ impl Withdrawal {
     }
 }
 
+/// Represents a collection of Withdrawals.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Hash, RlpEncodable, RlpDecodable)]
```

Comment:
this doesn't match what we have here:

https://github.com/paradigmxyz/reth/blob/ff90079f9186a1e4e4f755a363e51b92f0b0e9ad/crates/primitives-traits/src/withdrawal.rs#L26-L27

Reply:
Fixed thanks for pointing out